app/views/layouts/application.html.erb

```diff
@@ -20,64 +20,46 @@
+      <div class="nhsuk-header__container app-header__container">
         <div class="nhsuk-header__logo nhsuk-header__logo--only">
           <a class="nhsuk-header__link nhsuk-header__link--service"
              href="<%= @header_path %>"
              aria-label="Service homepage">
-            <svg class="nhsuk-logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 16" height="40" width="100">
+            <svg class="nhsuk-logo app-logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 16" height="40" width="100">
               <path class="nhsuk-logo__background" fill="#005eb8" d="M0 0h40v16H0z"></path>
               <path class="nhsuk-logo__text" fill="#fff" d="M3.9 1.5h4.4l2.6 9h.1l1.8-9h3.3l-2.8 13H9l-2.7-9h-.1l-1.8 9H1.1M17.3 1.5h3.6l-1 4.9h4L25 1.5h3.5l-2.7 13h-3.5l1.1-5.6h-4.1l-1.2 5.6h-3.4M37.7 4.4c-.7-.3-1.6-.6-2.9-.6-1.4 0-2.5.2-2.5 1.3 0 1.8 5.1 1.2 5.1 5.1 0 3.6-3.3 4.5-6.4 4.5-1.3 0-2.9-.3-4-.7l.8-2.7c.7.4 2.1.7 3.2.7s2.8-.2 2.8-1.5c0-2.1-5.1-1.3-5.1-5 0-3.4 2.9-4.4 5.8-4.4 1.6 0 3.1.2 4 .6"></path>
             </svg>
-            <span class="nhsuk-header__service-name">
+            <span class="nhsuk-header__service-name app-header__service-name">
               <%= @service_name %>
             </span>
           </a>
         </div>
-        <div class="nhsuk-header__content" id="content-header">
-
-          <div class="nhsuk-header__menu">
-            <button class="app-header__menu-toggle--no-search" id="toggle-menu" aria-controls="header-navigation" aria-expanded="false">Menu</button>
-          </div>
-        </div>
       </div>
 
-      <nav class="nhsuk-header__navigation" id="header-navigation" role="navigation" aria-label="Primary navigation" aria-labelledby="label-navigation">
-        <div class="nhsuk-width-container">
-          <p class="nhsuk-header__navigation-title"><span id="label-navigation">Menu</span>
-            <button class="nhsuk-header__navigation-close" id="close-menu">
-              <svg class="nhsuk-icon nhsuk-icon__close" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false" width="27" height="27">
-                <path d="M13.41 12l5.3-5.29a1 1 0 1 0-1.42-1.42L12 10.59l-5.29-5.3a1 1 0 0 0-1.42 1.42l5.3 5.29-5.3 5.29a1 1 0 0 0 0 1.42 1 1 0 0 0 1.42 0l5.29-5.3 5.29 5.3a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42z"></path>
+      <nav class="nhsuk-navigation" id="header-navigation" role="navigation" aria-label="Primary navigation">
+        <ul class="nhsuk-header__navigation-list app-header__navigation-list">
+          <% if current_user.present? %>
+            <li class="nhsuk-header__navigation-item">
+              <div class="app-header__navigation-text">
+                <%= current_user.email %>
+              </div>
+            </li>
+            <li class="nhsuk-header__navigation-item">
+              <%= button_to destroy_user_session_path, class: "app-button-link", method: :delete do %>
```

Comment:
Possible suggestion for the name we give to the sign  out button:

```suggestion
              <%= button_to destroy_user_session_path, class: "app-header__button-link", method: :delete do %>
```

Reply:
Good point, it's inextricably styled to be in the header.